CMakeLists.txt

```diff
@@ -203,38 +203,19 @@ if(ENABLE_IPV6 AND NOT WIN32)
   endif()
 endif()
 
-option(ENABLE_MANUAL "to provide the built-in manual" ON)
-unset(USE_MANUAL CACHE) # TODO: cache NROFF/NROFF_MANOPT/USE_MANUAL vars?
+CURL_NROFF_CHECK()
```

Comment:
Any reason why you converted it to a macro? It does not really seem fit for the existing macros.cmake file

Reply:
@Lekensteyn Any reason not to do so? Why does it not fit?  It checks for nroff presence and its capability to do the job. 
We can add additional FindNROFF module if you wish.
Or we can rename it to CHECK_CURL_NROFF as all macros in this file.

Reply:
A FindNROFF.cmake file would be better I think, my complaint is here because the other macros are genuine macros that take a parameter which is not the case with NROFF.